plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/iapi-frontend.ts

```diff
@@ -850,6 +846,77 @@ const { state: cartItemState } = store(
 					}
 				}
 			},
+			itemItemData() {
+				const { ref } = getElement();
+
+				if ( ! ref ) {
+					return;
+				}
+
+				const populateSingleItemData = (
+					element: Element,
+					item: CartItemDataAttr
+				) => {
+					const nameEl = element.querySelector(
+						'div span.wc-block-components-product-details__name'
+					);
+					const valueEl = element.querySelector(
+						'div span.wc-block-components-product-details__value'
+					);
+
+					const name = item?.key || item.name || '';
```

Comment:
I think the easiest win here as I mentioned before would be derived getters for these so this logic is not duplicated.